pilz_robots/package.xml

```diff
@@ -18,6 +18,7 @@
   <exec_depend>prbt_support</exec_depend>
   <exec_depend>prbt_ikfast_manipulator_plugin</exec_depend>
   <exec_depend>prbt_moveit_config</exec_depend>
+  <exec_depend>pilz_testutils</exec_depend>
```

Comment:
Should the meta-package really depend on the test_utils to install the package as dependency?
I'd prefer the package to be `test_depend` in the packages that need it.

Reply:
Done.